lms/assets.py

```diff
@@ -133,20 +133,20 @@ def wrapper(context, request):
     return wrapper
 
 
-def _load_bundles(fp):
+def _load_bundles(file_):
     """Read an asset bundle config from a file object."""
     parser = configparser.ConfigParser()
-    parser.read_file(fp)
+    parser.read_file(file_)
     return {k: aslist(v) for k, v in parser.items("bundles")}
 
 
 # Site assets
-ASSETS_VIEW = static_view("lms:../build", cache_max_age=None, use_subpath=True)
-ASSETS_VIEW = _add_cors_header(ASSETS_VIEW)
```

Comment:
PyLint doesn't like it when you reassign a variable (`ASSETS_VIEW`) to a different type. So I just combined these two into one line